### ISSUE TITLE
feat: submitBlock

### ISSUE THREAD

**Issue by BelfordZ** [paritytech/parity-ethereum#10784]
# New JSON-RPC method submitBlock

## Purpose

I would like to be able to submitWork, but have the address used be one other than the etherbase of the node the work is being submitted by.

This would allow signing of new blocks to occur outside of the client, similar to sendRawTransaction allows you to externally sign a tx. 

## Proposed solution

Named as per @sorpaas suggestion, perhaps this could be done as a `submitBlock`. 
It would look as follows:
![image](https://user-images.githubusercontent.com/1787231/60139004-334fcb00-9761-11e9-822f-244d360649e1.png)

Here is an OpenRPC describing the method doc if you want to [play](https://playground.open-rpc.org) with it.
```json
{
  "openrpc": "1.1.8",
  "info": {
    "version": "1.0.0",
    "title": "Example submitBlock"
  },
  "methods": [
    {
      "name": "parity_submitBlock",
      "summary": "Submit a new block proposal",
      "description": "Given a new block, it will validate the block, and broadcast it to connected peers.",
      "params": [
        { "$ref": "https://raw.githubusercontent.com/etclabscore/ethereum-json-rpc-specification/master/openrpc.json#/components/contentDescriptors/Block" }
      ],
      "result": {
        "name": "isBlockAccepted",
        "schema": {
          "type": "boolean"
        }
      }
    }
  ]
}
```

**Comment by seunlanlege** [comment]
this is already implemented in #10609

**Comment by BelfordZ** [comment]
great! thanks @seunlanlege n friends!
